### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by h-michael**
#462 add OS X desktop entry Makefile.

Thanks @brycefisher

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,6 +156,15 @@ sudo cp target/release/alacritty /usr/local/bin # or anywhere else in $PATH
 cp Alacritty.desktop ~/.local/share/applications
 ```
 
+#### OS X
+
+To install the desktop entry for Alacritty, run
+
+```sh
+make app
```
Comment (brycefisher) [inline README.md]: Yes and thanks for adding this to README!!

IIRC, the Makefile targets all require that you've already run `cargo build --release`, right?

Comment (jwilm) [inline README.md]: That's right, and I'd rather not put the `cargo build --release` into the Makefile since people may want to enable different features (once we have them *cough* scrollback). I think the thing to do here is just add it to the instructions.

**Comment by jwilm** [comment]
Thanks for the PR! Just need one little change, and then it's ready to go.

File: README.md (hunk 1 above)
Comment (brycefisher) [inline README.md]: Wait...@jwilm are you saying that scrollback is coming? I thought it was a tmux or bust kind of situation

Comment (jwilm) [inline README.md]: @brycefisher I'm leaning towards adding it behind a feature gate. Folks who don't want it can have it 100% compiled out and not affect perf, and those that are interested can opt-in. Rust's feature system would make it pretty easy.

Comment (brycefisher) [inline README.md]: Whoa! So exciting! I guess I'll stop learning tmux now

**Comment by h-michael** [comment]
@jwilm @brycefisher 
Thanks for your reviewing.
I updated instructions.

**Comment by brycefisher** [comment]
awesome, @h-michael! Thanks for updating the text.

I find the wording of the current PR a bit confusing. The Makefile does _not_ compile Alacritty for you. So, I might just rephrase this as:

> You can pack a desktop app for OS X by first running `cargo build --release`, then running this task:
>
> ```sh
> make install
> ```

The exact phrasing isn't super important, but clearly indicating that you have to invoke `cargo build` before the make target is the key point.

**Comment by h-michael** [comment]
Thanks @brycefisher 
I think that running cargo build is unnecessary, because Makfile includes 'binary' task, which runs `cargo build --release`.
So even if someone runs `make install` without `cargo build --release`, if need be, task runs `cargo build --release`.
Do I have a misunderstanding that you said ?

**Comment by brycefisher** [comment]
@h-michael apologies for the slow response. You understood me correctly and you are 💯 correct! Sorry to misunderstand the Makefile! LGTM.

**Comment by h-michael** [comment]
@brycefisher Thanks !!

@jwilm Please merge whenever it is convenient for you.

**Comment by jwilm** [comment]
There's some issues with the desktop entry that I'd like to resolve before publicizing it too much. Going to close this, and we can look at documenting it later.

Thanks for the PR @h-michael, and sorry for the change of heart about landing it.
